Sources/App/configure.swift

```diff
@@ -104,6 +104,10 @@ public func configure(_ app: Application) throws {
     do {  // Migration 022 - add is_archived to repositories
         app.migrations.add(UpdateRepositoryAddIsArchived())
     }
+    do {
```

Comment:
nit: do we want to continue the pattern of the migration number and comment here?